04_environment/infra.tex

```diff
@@ -0,0 +1,212 @@
+                                     & \texttt{Display}                                                              & \begin{tabular}[c]{@{}l@{}}Returns a JSON \\ string for a \\ foraging report\end{tabular}                                                                         & - Foraging Report                                                                                                                                           & - JSON String                                                                                                                                                               \\ \hline
+\multirow{3}{*}{\texttt{setup.go}}   & \texttt{CreateDeerHunt}                                                       & \begin{tabular}[c]{@{}l@{}}Receives hunt \\ participants and \\ their contributions \\ and returns a \\ DeerHunt\end{tabular}                                     & \begin{tabular}[c]{@{}l@{}}- Deer hunt \\ configuration \\ - Logger\end{tabular}                                                                            & \begin{tabular}[c]{@{}l@{}}- Deer hunt\\ - Error\end{tabular}                                                                                                               \\ \cline{2-5} 
+                                     & \texttt{\begin{tabular}[c]{@{}c@{}}CreateFishing\\ Expedition\end{tabular}}   & \begin{tabular}[c]{@{}l@{}}Receives hunt \\ participants and \\ their contributions \\ and returns a \\ FishHunt\end{tabular}                                     & \begin{tabular}[c]{@{}l@{}}- Island IDs with\\ their contributions\\ - Fish hunt \\ configuration \\ - Logger\end{tabular}                                  & \begin{tabular}[c]{@{}l@{}}- Fishing \\ Expedition\\ - Error\end{tabular}                                                                                                   \\ \cline{2-5} 
+                                     & \texttt{\begin{tabular}[c]{@{}c@{}}CreateDeer\\ PopulationModel\end{tabular}} & \begin{tabular}[c]{@{}l@{}}Returns the target\\ population model.\end{tabular}                                                                                    & \begin{tabular}[c]{@{}l@{}}- Deer hunt \\ configuration\\ - Logger\end{tabular}                                                                             & - Population model                                                                                                                                                          \\ \hline
+\end{tabular}
+\end{center}
+\caption{Foraging Main Functions}
+\label{tab:Foraging Main Functions}
+\end{table}
+
+\newpage
+\subsection{Important Implementation Remarks}
+\subsubsection{Utility for a Single Deer}
```

Comment:
This is repeated in the design environment section. May be worth removing or just referencing the other section

Reply:
Hey @NGovani 
Are you referring to the complete subsection or the subsubsection?
There are similar parts and the same equations as the design doc but we don't believe it should remove the entire subsection as for example deer population is described here.

Reply:
The subsubsections for utility is what I meant. It feels like it describes a similar process to what happens in tiers in design section of the environment along with the Bernoulli, if you think it helps to combine all of them into a single part (considering they are spread out in the design section) then it should be fine to keep.

Reply:
Hey, 
so after discussing a little bit more with the group, 
we do describe the same concepts as in the design doc but in wider details and depth and elaborate a lot more on the math. Therefore, they are similar but different. 
may we keep it as is and refer it to the design section?
the only prob is that the design doc has been already merged 
Let me know what you think : )

Reply:
> The subsubsections for utility is what I meant. It feels like it describes a similar process to what happens in tiers in design section of the environment along with the Bernoulli, if you think it helps to combine all of them into a single part (considering they are spread out in the design section) then it should be fine to keep.

Great! Thanks, we will remove the utility section and a few equations that are the same as the design doc  :)